packages/mds-stream/index.ts

```diff
@@ -17,8 +17,35 @@
 import logger from '@mds-core/mds-logger'
 import redis from 'redis'
 import bluebird from 'bluebird'
+import Cloudevent from 'cloudevents-sdk'
 import { Device, VehicleEvent, Telemetry } from '@mds-core/mds-types'
 
+const { env } = process
+const binding = null
+
+function getBinding() {
+  if (!binding) {
+    const config = {
+      method: 'POST',
+      url: env.SINK
+    }
+
+    // eslint-disable-next-line new-cap
+    return new Cloudevent.bindings['http-binary0.2'](config)
```

Comment:
Shouldn't there be a line like `binding = new Cloudevent.binds['http-binary0.2'](config)` to cache it?

Reply:
Why not? Changed in 5c0527e